experimental/table/server/under/hive/src/main/java/alluxio/table/under/hive/util/PathTranslator.java

```diff
@@ -65,7 +68,13 @@ public PathTranslator addMapping(String alluxioPath, String ufsPath) {
   public String toAlluxioPath(String ufsPath) throws IOException {
     for (BiMap.Entry<String, String> entry : mPathMap.entrySet()) {
       if (ufsPath.startsWith(entry.getValue())) {
-        return entry.getKey() + ufsPath.substring(entry.getValue().length());
+        String alluxioPath = entry.getKey() + ufsPath.substring(entry.getValue().length());
+        if (alluxioPath.startsWith("/")) {
+          // scheme/authority are missing, so prefix with the scheme and authority
+          alluxioPath =
+              ConfigurationUtils.getSchemeAuthority(ServerConfiguration.global()) + alluxioPath;
```

Comment:
Do we do this often, it may be a useful utils method

Reply:
It only happens here in the translation, and the translation only happens once per table/partition load.